docs/Tutorials/Generating-Certificates.md

```diff
@@ -0,0 +1,126 @@
+description: Generate Orion Certificates 
+<!--- END of page meta data -->
+
+# Generating certificates
+
+You can generate certificates using Orion, or a third-party tool like OpenSSL or
+Keytool.
+
+## Generating certificates using Orion
+
+You can use Orion to generate certificates between Orion nodes. Start the nodes
+with the TLS trust mode set to `insecure-no-validation` for `tlsclienttrust` 
+and `tlsservertrust` until the `tlsknownclients` and `tlsknownservers` files are
+populated. When the files are populated, restart the nodes with the TLS trust
+mode set to `whitelist` for `tlsclienttrust` and `tlsservertrust`.
```

Comment:
Can you also do the equivalent thing to use Orion to generate certificates for the Besu-Orion connection?  Or can you only use Orion to generate certificates for the Orion-Orion connection? Either way, I think this needs to be made clear

Reply:
Done